.travis.yml

```diff
@@ -30,7 +30,7 @@ before_install:
 install:
 
   - if [[ $MINIMAL_ENV == 'False' ]] ; then
-      DEPS="numpy scipy matplotlib ipython h5py sympy scikit-learn dill natsort setuptools scikit-image cython lxml ipyparallel";
+      DEPS="pytest pytest-cov numpy scipy matplotlib ipython h5py sympy scikit-learn dill natsort setuptools scikit-image cython lxml ipyparallel dask toolz";
```

Comment:
that's right, no longer required. Though `dask` uses it extensively, so will be pulled in any way. Will get rid of it on our side, though